libs/comparator/tables.js

```diff
@@ -39,6 +39,13 @@ const buildColumnsMap = table => {
                 column.unique && (options.push('UQ'))
                 column.required && (options.push('NN'))
 
+                if (relation.metaInfo?.relationIdentificationColumnId) {
+                    column.metaInfo || (column.metaInfo = {})
+                    column.metaInfo.relationIdentificationColumnId = relation.metaInfo.relationIdentificationColumnId
+
+                    options.push(relation.metaInfo.relationIdentificationColumnName)
```

Comment:
please attach a screenshot of how the discrepancy will be reported

Reply:
<img width="838" alt="image" src="https://github.com/Carrydmcc/bact/assets/68130217/48ca21e8-5b75-4d72-a9e4-0a5d0fbb64d5">